doc/appendix-de-kassensichv/cash-register-integration/cash-register-integration.md

```diff
@@ -1,58 +1,58 @@
 ## Cash Register Integration
 
-This chapter describes the cash register integration in accordance with the German law. The general rules for cash register integration are described in General Part of this document.
+This chapter describes the cash register integration in accordance with German law. The general rules for cash register integration are described in General Part of this document.
 
 ### Receipt Creation Process
 
-This chapter describes the general process of creating receipts with fiskaltrust.Middleware and its workflow, in accordance with the German law. It requires to give a scope to an ongoing action over time. This scope is named a transaction. Calls to fiskaltrust.Middleware are processed just in time and cannot be async over multiple minutes. Therefore and in accordance with German law, a single call is maybe not able to scope a complete transaction. To solve this, multiple calls are used, scoping the same transaction.
+This chapter describes the general process of creating receipts with fiskaltrust.Middleware and its workflow, following German law. It requires to give a scope to an ongoing action over time. This scope is named a transaction. Calls to fiskaltrust.Middleware are processed just in time and cannot be async over multiple minutes. Therefore and in accordance with German law, a single call is maybe not able to scope a complete transaction. To solve this, multiple calls are used, scoping the same transaction.
```

Comment:
This scope is named a transaction. Calls to fiskaltrust.Middleware are processed just in time and cannot be async over multiple minutes. >> I don understand, the name of the scope described is "transaction"?.
The part over multiple minutes is not very clear to me either. It meant that they cannot be processed asynchronously? If so, the "async over multiple minutes" could be replaced with  "cannot be processed asynchronously". besides, async is not a verb or an adjective!

Reply:
i also do not understand this part...